src/LargeXlsx/XlsxWriter.cs

```diff
@@ -178,14 +178,14 @@ private void SaveWorkbookRels()
             }
         }
 
-        public XlsxWriter BeginWorksheet(string name, int splitRow = 0, int splitColumn = 0, bool rightToLeft = false, IEnumerable<XlsxColumn> columns = null)
+        public XlsxWriter BeginWorksheet(string name, int splitRow = 0, int splitColumn = 0, bool showGridLines = true, bool showHeaders = true, bool rightToLeft = false, IEnumerable<XlsxColumn> columns = null)
```

Comment:
In order to preserve backward API compatibility, for those using passing arguments positionally, we should put the new parameters at the end of the function signature, after `columns`. I know I've been guilty of this myself in past versions, and that it "aesthetically" it would made more sense the way you put them, but I feel like preserving API compatibility is better at the moment.